cobalt/css_parser/BUILD.gn

```diff
@@ -100,7 +100,7 @@ static_library("css_parser") {
     "//starboard/common",
   ]
 
-  if (is_clang_16) {
+  if (is_clang && current_os != "android") {
```

Comment:
Don't use `current_os == "android"` (or the negation), use the variable `is_android`

Reply:
done